Cython/Build/Tests/TestCyCache.py

```diff
@@ -1,15 +1,20 @@
-import difflib
 import glob
 import gzip
 import os
 import tempfile
 
-import Cython.Build.Dependencies
-import Cython.Utils
-from Cython.TestUtils import CythonTest, write_file, fresh_cythonize
+from Cython.TestUtils import (CythonTest, write_file, fresh_cythonize,
+                              relative_lines_from_file)
+
+SAME = "The result of cytonization is the same"
```

Comment:
Typo
```suggestion
SAME = "The result of cythonization is the same"
```

Reply:
same in the TestRecythonize, i will fix it there